cypress/e2e/signUp.cy.js

```diff
@@ -1,5 +1,16 @@
+/// <reference types="cypress" />
+const { generateUser } = require('../support/generate');
 describe('Sign Up page', () => {
+  before(() => {
+    const URL_SIGN_UP = 'https://react-redux.realworld.io/#/register?_k=usft36';
+    cy.visit(URL_SIGN_UP);
+  });
   it('should provide an ability to register new account', () => {
-
+    const user = generateUser();
```

Comment:
OR const { username, email, password } = generateUser() and you haven't call object user